DEVDOC.md

```diff
@@ -181,6 +187,9 @@ dlv exec --headless --listen=127.0.0.1:1234 --api-version 2 ./bazel-bin/pkg/gapi
 
 6. Now VSCode can interact with Delve and can be used for debugging `gapis` in VSCode UI instead of the command line. Enjoy your debugging :)
 
+This will allow you to put breakpoint in any lines in Go standard library, generated or handwritten AGI Go source code. For the generated file,
```

Comment:
Present tense, and remove a few "the" and some typos:

```
This allows you to put breakpoints at any line in the Go standard library, or in
generated or handwritten AGI Go source code. For generated files, you can put
breakpoints under the  `bazel-bin/` or `bazel-out/` folder and the debugger will
still find the relevant files in the gofuse directory. The only downside is that you
may have two versions of the same file, but this is the only working workaround
until VSCode Go Plugin properly supports Bazel-generated files.
```

(Feel free to adapt as you see fit)